e2e-tests/one-pod/run

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+spinup_psmdb() {
+    local cluster=$1
+    local config=$2
+    local size="${3:-3}"
+
+    desc 'create first PSMDB cluster'
+    apply_cluster $config
+
+    desc 'check if Pod is started'
+    wait_for_running "${cluster}" "$size"
+    sleep 20
```

Comment:
I think sleep not needed here
but I want to control that TLS and encryption are disabled, so it is needed to compare statefulset content
```suggestion
    compare_kubectl statefulset/$cluster
```

Reply:
is sleep really needed?

Reply:
it takes time to make new mongo user available inside MongoDB. If we remove sleep we'll face auth error.